presto-parquet/pom.xml

```diff
@@ -158,6 +223,16 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.basepom.maven</groupId>
```

Comment:
Why don't we move this to the other commit?

Reply:
Done.